source/Ship.cpp

```diff
@@ -4258,6 +4260,20 @@ const vector<weak_ptr<Ship>> &Ship::GetEscorts() const
 
 
 
+int Ship::GetLingerSteps() const
+{
+	return lingerSteps;
+}
+
+
+
+void Ship::Linger()
+{
+	lingerSteps++;
```

Comment:
I think we usually do prefix.
```suggestion
	++lingerSteps;
```